### DISCUSSION TITLE
ISAE 3402 Template

### DISCUSSION THREAD

**Discussion by drummbelbummel** [intuitem/ciso-assistant-community#2880] [Ideas]
**Problem statement**
I want to be able to import ISAE 3402 Controls.
Is there any template for this?
Thx.

**Comment by ab-smith** [comment]
Hello @drummbelbummel 
As a matter of fact, SOC 2 is part of the supported libraries but not SOC 1. Is this your request?
Regards,

**Comment by KlavsKlavsen** [comment]
as I read this - https://linfordco.com/blog/soc-2-isae-3402-aicpa-attestation-standards/ - soc 1 is standard that lines up with the requirements of isae-3402 and soc 2 lines up with the requirements for isae 3000. Is this correct? 
Which means the SOC 2 library should also contain isae-3000 requirements?
